tests/test_gen_linear.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+PyTest file for linear Fresnel
+This is intended to be run with PySAM 1.2.1
+
+Created on 2/6/2020
+@author: Mike Bannister
+"""
+
+import os
+import logging
+import pytest
+import numpy as np
+
+from reV.generation.generation import Gen
+from reV import TESTDATADIR
+
+
+def test_gen_linear(caplog):
+    """Test generation for linear Fresnel"""
+
+    caplog.set_level(logging.DEBUG)
```

Comment:
what is caplog? Typically we don't actively log in tests.

Reply:
Removed